### PR TITLE
Fibonacci homework

### PR DESCRIPTION
**Pull request by wbrejecki**

### CODE REVIEW

File: module1/homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
+		return 0;
+
+	if (sequence == 1)
+		return 1;
+
+
+	int first = 0, second = 1;
+	int result = 0;
+
+	for(int k = 2; k <= sequence; ++k)
```
Comment (kuchichan) [inline module1/homework/fibonacci/fibonacci.hpp:15]: add space space between  for `for (int k = 2 ..)`

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+
+   	if (sequence < 1)
+		return 0;
+
+	if (sequence == 1)
+		return 1;
+
```
Comment (kuchichan) [inline module1/homework/fibonacci/fibonacci.hpp:10]: One empty line is enough :)

File: module1/homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,37 @@
...
-    return 0;
+    if (sequence==0)
+    {
+        return 0;
+    }
+    if (sequence==1)
+    {
+        return 1;
+    }
+    return (fibonacci_recursive(sequence-1)+fibonacci_recursive(sequence-2));
```
Comment (kuchichan) [inline module1/homework/fibonacci/fibonacci.hpp:35]: Spaces between `+` operator

File: module1/homework/fibonacci/fibonacci.hpp (hunk 2 above)
Comment (nauka-programowania-MA) [reply module1/homework/fibonacci/fibonacci.hpp:10]: and simplify fo if (sequence <=1) return sequence;

Comment (wbrejecki) [reply module1/homework/fibonacci/fibonacci.hpp:10]: @nauka-programowania-MA What difference does it make?

Comment (nauka-programowania-MA) [reply module1/homework/fibonacci/fibonacci.hpp:10]: For 0 return - for 1 return 1 :) this value you have already in sequence variable

Comment (wbrejecki) [reply module1/homework/fibonacci/fibonacci.hpp:10]: You're clearly right, thanks! :)

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1430], "points": [2, 6] }

**Pull request closed without merging**